exercises/concept/vehicle-purchase/.docs/hints.md

```diff
@@ -0,0 +1 @@
+# Hints
```

Comment:
Maybe worth mentioning `min` here to help with `choose_vehicle()`?

(I assume you're avoiding `sort()` since then one has to address vector subsetting).

Reply:
Depends on the main discussion, about which order the exercises are presented.